### PR TITLE
Frontend Update

### PR DESCRIPTION
**Pull request by Rye-Guy**

### CODE REVIEW

File: backend/app/controllers/organization.controller.js [hunk 1]
```diff
@@ -28,7 +28,11 @@ exports.login = async (req, res) => {
         let authenticateOrganization = await organization.authenticate(email, password);
         await organization.update({logged_in: true}, {where: { email } });
         res.cookie('auth_token', authenticateOrganization.authToken.token, { expires: new Date(Date.now() + 54000000)})
-        return res.send(`${email} Successfully Authenticated`);
+        return res.json({
+            "message": `${email} Successfully Authenticated`,
```
Comment (lucazeni) [inline backend/app/controllers/organization.controller.js:32]: when you get a chance lets remove this commit from this PR, i will add it to the one i opened

File: backend/app/controllers/user.controller.js [hunk 2]
```diff
@@ -85,18 +85,19 @@ exports.getUsersByOrg = async (req, res) => {
 
     const orgEmail = req.body.organizationEmail;
     const authorizationHeader = req.headers.authorization;
+    
     // check if correct body was sent in the request
-    if (!orgEmail) {
-        return res.status(400).send(JSON.parse('{"Required-Body-Error":{"organizationEmail": "string"}}'));
-    }
+    // if (!orgEmail) {
```
Comment (lucazeni) [inline backend/app/controllers/user.controller.js:90]: same with this one

File: backend/app/controllers/organization.controller.js (hunk 1 above)
Comment (Rye-Guy) [reply backend/app/controllers/organization.controller.js:32]: Ok will do.

File: backend/app/controllers/user.controller.js (hunk 2 above)
Comment (Rye-Guy) [reply backend/app/controllers/user.controller.js:90]: kk

**Pull request merged**